src/Core/Waiter.php

```diff
@@ -114,14 +114,15 @@ public function __destruct()
      * Make sure the actual request is executed.
      *
      * @param float|null $timeout Duration in seconds before aborting. When null wait until the end of execution.
+     * > When null wait until the end of execution => not what the code does, isn't it? See proposal in Result.php
```

Comment:
Should be `wait until the arrival of the first bits`?

Reply:
the client is not used at all right if the timeout is null - I fail to understand how this can doanything useful...

Reply:
ohh.. good catch